cmd/ethrex/cli.rs

```diff
@@ -37,9 +38,10 @@ pub struct Options {
         help = "Receives a `Genesis` struct in json format. This is the only argument which is required. You can look at some example genesis files at `test_data/genesis*`.",
         long_help = "Alternatively, the name of a known network can be provided instead to use its preset genesis file and include its preset bootnodes. The networks currently supported include holesky, sepolia, hoodi and mainnet.",
         help_heading = "Node options",
-        env = "ETHREX_NETWORK"
+        env = "ETHREX_NETWORK",
+        value_parser = clap::value_parser!(Network),
     )]
-    pub network: Option<String>,
+    pub network: Option<Network>,
```

Comment:
nit: instead of an `Option`, it would be nice to have a default (same as we have with `evm` arg). The default should be mainnet